app/lucy/src/app/directives/click-away/click-away.directive.spec.ts

```diff
@@ -2,7 +2,7 @@ import { ClickAwayDirective } from './click-away.directive';
 
 describe('ClickAwayDirective', () => {
   it('should create an instance', () => {
-    const directive = new ClickAwayDirective();
+    const directive = new ClickAwayDirective(undefined);
```

Comment:
why explicitly setting undefined

Reply:
I forgot to change this one actually, will do that 👍